src/components/AddCustomerForm.jsx

```diff
@@ -0,0 +1,123 @@
+import React, { useState } from "react";
+import "./AddCustomerForm.css";
+
+const AddCustomerForm = props => {
+  const initUser = {
+    id: null,
+    title: "",
+    firstName: "",
+    surname: "",
+    email: "",
+    roomId: " ",
+    checkInDate: "",
+    checkOutDate: ""
+  };
+
+  const [customer, setCustomer] = useState(initUser);
+
+  const handleChange = event => {
+    const updateCustomer = {
+      ...customer,
+      [event.target.name]: event.target.value
+    };
+    setCustomer(updateCustomer);
+  };
+
+  const handleSubmit = event => {
+    event.preventDefault();
+    console.log(customer.email);
+    if (
+      customer.email &&
+      customer.firstName &&
+      customer.surname &&
+      customer.title &&
+      customer.roomId &&
+      customer.checkInDate &&
+      customer.checkOutDate
+    ) {
+      handleChange(event, props.addCustomer(customer));
+    }
+  };
+
+  return (
+    <form class="form-group">
```

Comment:
If you create a form element then you need to attach the `handleSubmit` event handler function to the form element

Reply:
You've attached it to a div  below

Reply:
Aah that was by mistake thats why I was getting some strange behaviour.